src/v1/authentication/__tests__/routers/authentication/login.js

```diff
@@ -41,22 +48,38 @@ describe(`Test the ${ROUTE} path`, () => {
     const { auth, message } = result.body;
 
     expect(auth).toBe(false);
-    expect(message).toBe('Password is wrong');
+    expect(message).toMatchSnapshot();
   });
 
   it('should not login user, because password is missing', async () => {
     const body = { username };
     const result = await simulate.post(ROUTE, 400, body);
     const { message } = result.body;
 
-    expect(message).toBe('All fields required.');
+    expect(message).toMatchSnapshot();
   });
 
   it('should not login user, because username is missing', async () => {
     const body = { password };
     const result = await simulate.post(ROUTE, 400, body);
     const { message } = result.body;
 
-    expect(message).toBe('All fields required.');
+    expect(message).toMatchSnapshot();
+  });
+
+  it('should not login user, because username too short', async () => {
+    const body = { usernahe: shortUsername, password };
+    const result = await simulate.post(ROUTE, 400, body);
+    const { message } = result.body;
+
+    expect(message).toMatchSnapshot();
+  });
+
+  it('should not login user, because password too short', async () => {
+    const body = { username, password: shortPassword };
+    const result = await simulate.post(ROUTE, 400, body);
```

Comment:
You only assert on body, no other data from the result is needed, so simplify it within `simulate`.
You will avoid destructuring in a lot of places.